tests/test-cases/Group1-Docker-Commands/1-05-Docker-Start.robot

```diff
@@ -31,15 +31,17 @@ Simple start
     Should Not Contain  ${output}  Error:
 
 Start with attach and interactive
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
-    Should Be Equal As Integers  ${rc}  0
-    Should Not Contain  ${output}  Error
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -it busybox /bin/top
-    Should Be Equal As Integers  ${rc}  0
-    Should Not Contain  ${output}  Error:
-    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start -ai ${output}
-    Should Be Equal As Integers  ${rc}  0
-    Should Not Contain  ${output}  Error:
+    ${status}=  Get State Of Github Issue  4634
+    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-05-Docker-Start.robot needs to be updated now that Issue #4634 has been resolved
+    # ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} pull busybox
+    # Should Be Equal As Integers  ${rc}  0
+    # Should Not Contain  ${output}  Error
+    # ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} create -it busybox /bin/top
+    # Should Be Equal As Integers  ${rc}  0
+    # Should Not Contain  ${output}  Error:
+    # ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} start -ai ${output}
+    # Should Be Equal As Integers  ${rc}  0
+    # Should Not Contain  ${output}  Error:
```

Comment:
This test will never be supported in CI, it needs to be moved to manual/Group17-Manual-TTY pleaes